tests/test_shardformer/test_model/_utils.py

```diff
@@ -23,6 +22,11 @@
 from colossalai.tensor.d_tensor.api import is_customized_distributed_tensor, is_distributed_tensor
 
 
+def print_rank(prompt, value, rank=0):
```

Comment:
colossalai.logging and coordinator can implement this feature

Reply:
fixed